client-websocket/build.gradle.kts

```
@@ -0,0 +1,22 @@
+// In this section you declare the dependencies for your production and test code
+dependencies {
+	// Common
+	api(project(":twitch4j-util"))
+
+	// WebSocket
+	api(group = "com.neovisionaries", name = "nv-websocket-client")
```

Since the details of the websocket implementation are not exposed to the user, we *could* remove `api` here (which is one less thing to worry about if we ever decide to switch ws libraries)
```suggestion
	implementation(group = "com.neovisionaries", name = "nv-websocket-client")
```